frontend/src/views/AccountView.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="flex-grow text-center">
+    <div class="text-green-500 italic" v-if="showerror">
+      Account nicht verfügbar
+    </div>
+  </div>
+  <div class="w-full mt-auto mb-auto ml-3 mr-3">
+    <AccountDetailsEditContainer
+      :showadminview="true"
```

Comment:
Shouldnt this be false in the account view?
Or at least for non admins

Reply:
see above